src/main/java/io/vertx/core/json/JsonArray.java

```diff
@@ -117,6 +117,17 @@ public String getString(int pos) {
     }
   }
 
+  /**
+   * Get the Integer at position {@code pos} in the array,
+   *
+   * @param pos the position in the array
+   * @return the Number, or null if a null value present
+   * @throws java.lang.ClassCastException if the value cannot be converted to Number
```

Comment:
```suggestion
   * @throws java.lang.ClassCastException if the value is not a Number
```
There is no conversion, only casting.